src/index.js

```diff
@@ -6,8 +6,8 @@ import axios from "axios";
 import { Provider } from "react-redux";
 import configureStore from "./state/store/configureStore";
 
-axios.defaults.baseURL = "http://localhost:3000/api/v1/";
-// axios.defaults.baseURL = "https://the-helping-hand.herokuapp.com/api/v1/";
+// axios.defaults.baseURL = "http://localhost:3000/api/v1/";
+axios.defaults.baseURL = "https://the-helping-hand.herokuapp.com/api/v1/";
```

Comment:
You can do something like this : 
```
axios.defaults.baseURL =
  process.env.NODE_ENV === "development"
    ? process.env.DEV_API_URL
    : "process.env.PROD_API_URL
```

Reply:
We will make usage of that in the next PR, thank you!